common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -38,10 +39,14 @@
 public class WynncraftButtonFeature extends Feature {
     private static final String GAME_SERVER = "play.wynncraft.com";
     private static final String LOBBY_SERVER = "lobby.wynncraft.com";
+    private boolean hasInitBootPased = false;
```

Comment:
Not only this variable has a spelling mistake, it is impossible to figure out what it means.

Please rename this to `firstTitleScreenInit` or something with the same meaning

Reply:
got it, using firstTitleScreenInit now, its a good name.
also sorry i just missed that spelling error somehow.